reference_submissions/criteo1tb/criteo1tb_jax/submission.py

```diff
@@ -101,13 +101,14 @@ def update_params(workload: spec.Workload,
   del current_params_types
   del loss_type
   del eval_results
-  del global_step
+  # del global_step
 
   optimizer_state, opt_update_fn = optimizer_state
   per_device_rngs = jax.random.split(rng, jax.local_device_count())
-  new_model_state, new_optimizer_state, new_params = pmapped_train_step(
+  loss, new_model_state, new_optimizer_state, new_params = pmapped_train_step(
       workload, opt_update_fn, model_state, optimizer_state,
       current_param_container, batch, per_device_rngs)
+  print(f'Step {global_step} loss = {loss[0]}')
```

Comment:
Should probably be deleted after debugging.

Reply:
lol thanks for catching this